packages/examples/kitchen-sink/src/index.js

```diff
@@ -1,5 +1,6 @@
 import styles from './styles.css';
 import parcel from 'url:./parcel.webp';
+import parcelFront from 'webp:./parcel-front.png';
```

Comment:
What ends up happening here? Is `parcelFront` a url? is the image inlined? realized we don't allow multiple named pipelines e.g. `url:webp:./parcel-front.png`, which seems like it might be problematic?

Reply:
I mentioned it in my comment above, I had to add this to make it work at all `"*.webp": ["@parcel/transformer-raw"]`
I think it should return a url?
Although it currently returns an empty object which seems weird to me?

Not entirely sure if we should allow chaining although I don't really have a better idea.